manifests/istio-control/istio-discovery/files/gen-istio.yaml

```diff
@@ -1711,7 +1714,7 @@ spec:
       patch:
         operation: INSERT_BEFORE
         value:
-          name: envoy.filters.http.wasm
+          name: stats
```

Comment:
what do you think about using something like `istio_stats` or similar to indicate that this filter is istio-specific?

Reply:
Sure.